### PR TITLE
The Times clone created by Uchenna and henoks with HTML and CSS3

### PR DESCRIPTION
**Pull request by uchennaanya**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,868 @@
...
+          <small>
+            <a href="#">Mobile Applications</a>
+            <a href="#">Replica Edition</a>
+            <a href="#">Español</a>
+            <a href="#">中文网</a>
+          </small>
+        </div>
+      </footer>
+    </div>
+    <footer class="last_footer">
```
Comment (expjazz) [inline index.html:850]: - [ ] The footer needs to have a fixed position.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,868 @@
...
+      <p>
+        <q>So we will need to wait and see before we jump up and down,</q>
+        Dr. Krauss said.
+      </p>
+      <p>
+        Corroboration might not be long in coming. The Planck spacecraft will
+        report its own findings this year. At least a dozen other teams are
+        trying similar measurements from balloons, mountaintops and space.
+      </p>
+      <h2>Spirals in the Sky</h2>
```
Comment (expjazz) [inline index.html:422]: - [ ] Those titles need more padding.

| Yours | Original |
|:---:|:---:|
|image | image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

### Great Work on this project :100: 

Your page looks nearly perfect. However, we still have a few details that we need to improve:

- [ ] You should not close your first Pull Request after the first review. You need to keep using the same (not open a new one every time). Kindly use [this](https://github.com/uchennaanya/newyork-times/pull/1) one.

- [ ] Your Pull Request summary is empty. You should write the features you are implementing in this project.

- [ ] You have a couple of improvements needed in the design. Kindly check the inline comments below for more detail.

#### As stated above, the project looks really good.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by henatan99** [comment]
This pull request is closed

**Pull request closed without merging**